src/compiler/parser/index.js

```diff
@@ -76,12 +76,9 @@ export function parse (
 
   function endPre (element) {
     // check pre state
-    if (element.pre) {
+    if (element.pre || platformIsPreTag(element.tag)) {
       inVPre = false
     }
-    if (platformIsPreTag(element.tag)) {
-      inPre = false
```

Comment:
@javoski 
As far I can see this, these are not the same. One is `inVPre`, the other `inPre`